editor/editor_help.cpp

```diff
@@ -1578,18 +1578,36 @@ void EditorHelp::_add_text(const String &p_bbcode) {
 	_add_text_to_rt(p_bbcode, class_desc);
 }
 
-void EditorHelp::generate_doc() {
-	doc = memnew(DocTools);
-	doc->generate(true);
+Thread EditorHelp::thread;
+
+void EditorHelp::_wait_for_thread() {
+	if (thread.is_started()) {
+		thread.wait_to_finish();
+	}
+}
+void EditorHelp::_gen_doc_thread(void *p_udata) {
```

Comment:
Add a blank line between function declarations:

```suggestion
}

void EditorHelp::_gen_doc_thread(void *p_udata) {
```